ocs_ci/ocs/cluster.py

```diff
@@ -239,7 +298,59 @@ def cluster_health_check(self, timeout=None):
             raise exceptions.CephHealthException("Cluster health is NOT OK")
 
         self.noobaa_health_check()
-        # TODO: OSD and MGR health check
+
+        try:
+            self.osd_health_check(expected_osd_count)
+        except exceptions.OsdCountException as e:
+            logger.error(e)
+            raise exceptions.CephHealthException("Cluster health is NOT OK")
```

Comment:
I'm thinking of some improvement that will help to debug faster:
1) add the exception (e) to the raise message - f"Cluster health is NOT OK - {e}"
2) In all X_health_check() methods, add the exception object 'e' to the exception message

In that way, it will be presented very clear in Jenkins without need to go to logs

Reply:
Done